src/core/search/vector_utils.cc

```diff
@@ -34,7 +35,7 @@ __attribute__((optimize("fast-math"))) float CosineDistance(const float* u, cons
   }
 
   if (float denom = sum_uu * sum_vv; denom != 0.0f)
-    return sum_uv / sqrt(denom);
+    return 1 - sum_uv / sqrt(denom);
```

Comment:
I've been wondering why I'm getting crappy results with cosine ~~similarity~~ distance, and here it was.. at the very core 🙂